pkg/reader/ccip.go

```diff
@@ -234,9 +234,8 @@ func (r *ccipChainReader) queryCommitReports(
 // processCommitReports decodes the commit reports from the query results
 // and returns the ones that can be properly parsed and validated.
 func (r *ccipChainReader) processCommitReports(
-	iter []types.Sequence, ts time.Time, limit int,
```

Comment:
`ccipChainReader` has a logger already, do we need to pass it via args?

Reply:
Yeah, the one passed in via args has the call context (sequence number, ocr phase)

Reply:
(makes debugging easier)